### ISSUE TITLE
Theme Assistant Fatal Error

### ISSUE THREAD

**Issue by daveaiello** [After6Services/mt-plugin-theme-assistant#2]
@BillPerrin reported the following error appeared in the Movable Type Activity Log when he created a few Movable Type websites for the purpose of testing Theme Assistant: 

> Theme Assistant died with: Can't use string ("$Smartphone::Smartphone::CMS::Li"...) as a HASH ref while "strict refs" in use at lib/MT.pm line 426.

Is this related to #1 ?

**Comment by daveaiello** [comment]
A possible fix for this issue, at least temporarily, is to disable the SmartphoneOption plugin.

When I did this on mttest.after6services.com, this error no longer appeared in the Movable Type Activity Log.

Now we need to see if the version of Theme Assistant that we created several weeks ago will work in an environment where we are trying to reapply our theme to an existing website.

**Comment by BillPerrin** [comment]
I don't believe any users even know that a smartphone option is available on MT. It should be OK to disable this plugin as long as we note that we turned it off...just in case someone eventually wants to use it. I don't foresee that day, but you never know.
